src/ipns/ipns-record.md

```diff
@@ -259,9 +285,91 @@ Finally, the network nodes may also republish their records, so that the records
 
 ### Record Creation
 
-IPNS record MUST be serialized as `IpnsEntry` protobuf, and `IpnsEntry.data` MUST be signed using the private key.
+IPNS record MUST be serialized as `IpnsEntry` protobuf, and `IpnsEntry.data`
```

Comment:
Maybe we could rename `IpnsEntry` to `IpnsRecord`?

Reply:
Not against it, but want to limit what we change in IPIP to minimum, so it is easier to review.
Let's make a separate editorial PR after this IPIP lands.